### ISSUE TITLE
can't load values.yaml for prometheus  

### ISSUE THREAD

**Issue by cyxinda** [kubevela/kubevela#6038]
**Describe the bug**
when I install the kube-prometheus-stack using helm , It is difficult to load values.yaml . It will be waitting on the page of velaux for a long long time.
![1111](https://github.com/kubevela/kubevela/assets/7237898/eff5d3f8-0f0d-46cb-a38e-8b073f6b750f)

**To Reproduce**
 

**Expected behavior**
I expect velaux page can load values.yaml very fast.
 

**KubeVela Version**
v1.8.2

**Cluster information**
 - Kubernetes Version [v1.24.10]

**Comment by chivalryq** [comment]
This is really a huge value.yaml. https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/values.yaml

If you could, please provide related apiserver logs and network inspection in your browser.

**Comment by cyxinda** [comment]
> This is really a huge value.yaml. https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/values.yaml
> 
> If you could, please provide related apiserver logs and network inspection in your browser.

yes,it is real.
the request content as follow:
```bash
curl 'http://192.168.5.14:32083/api/v1/repository/chart/versions?repoUrl=https:%2F%2Fprometheus-community.github.io%2Fhelm-charts&repoType=helm&chart=kube-prometheus-stack&project=default' -H 'User-Agent: Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:109.0) Gecko/20100101 Firefox/113.0' -H 'Accept: application/json' -H 'Accept-Language: zh-CN,zh;q=0.8,zh-TW;q=0.7,zh-HK;q=0.5,en-US;q=0.3,en;q=0.2' -H 'Accept-Encoding: gzip, deflate' -H 'Authorization: Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VybmFtZSI6ImFkbWluIiwiZ3JhbnRUeXBlIjoiYWNjZXNzIiwiZXhwIjoxNjg1MDk4MDAxLCJpc3MiOiJ2ZWxhLWlzc3VlciIsIm5iZiI6MTY4NTA5NDQwMX0.s3prSBJhDPZMmAdjMG2Wum24uFoNj5SJKWi-eVJzEHE' -H 'Connection: keep-alive' -H 'Referer: http://192.168.5.14:32083/applications'
```
The response to the above request is indeed large. And in fact, it was discontinued.
![222](https://github.com/kubevela/kubevela/assets/7237898/40ff8b2c-d9f0-4355-bc18-ad8d64eea90a)
I noticed that this request doesn't come with a version parameter.
And when I choose a special version on the page of velaux,there are no new request ,too.
So I think whether It could be improved in the interact behivor on the page of velaux to reduce the amount of data transferred per request.

**Comment by karl-chanel** [comment]
hey i  experience the same problem   when i try to install apisix with helm  , It takes a long time to render and sometimes fails.

**Comment by Somefive** [comment]
The prometheus stack helm chart uses a really long values file. It leaves huge burden on the UI render in the browser.

**Comment by karl-chanel** [comment]
I experience problem with apisix helm chart ,  When I try to change a value  And make a new deploy ,th ui always stuck. Sometimes long, sometimes short sometimes fails. So there is no caching ? Every time I try to change a single value, it will fetch the whole yaml again?

**Comment by Somefive** [comment]
Hmm, I'm not sure. I do not have too much knowledge of how the velaux code is written. But if it does have no caching at the browser side, there might be performance issues I think.

**Comment by karl-chanel** [comment]
I found this issue may be related to that you are using a 3rd party helm repo, Try using the  built in bitnami maybe help

**Comment by cyxinda** [comment]
> The prometheus stack helm chart uses a really long values file. It leaves huge burden on the UI render in the browser.

yes,it is true the values.yaml of prometheus helm chart so long .

**Comment by briankane** [comment]
This issue has been inactive for over 2 years and is scheduled for closure as part of our backlog maintenance.

  **If this issue is still relevant:**
  - Please comment below to keep it open
  - Provide updated context if the original description is outdated

  **Timeline:** If there's no activity by **Friday, October 24th**, this issue will be automatically closed.

  **Note:** Issues can always be reopened if they become relevant again. We encourage the community to reopen and reprioritize any incorrectly 
  closed issues.

**Comment by briankane** [comment]
This issue has been closed following the stale issue notice. 

If this issue is still relevant and should remain open, please:
1. Reopen the issue
2. Remove the 'stale' label
3. Provide an update on the current status or importance

Thank you for your understanding and contribution to KubeVela!
